docs/polaris/applications-and-libraries/libraries/spack-pe.md

```diff
@@ -3,6 +3,9 @@
 Spack is an HPC-oriented package manager which ALCF uses to install software for
 the user environment.
 
+Users should not depend on libraries present in the system, but instead load
+them from `/soft` using Spack as described below.
```

Comment:
I would say use libraries from the Spack PE _when possible_, so something more like
```
Users should depend on libraries in the Spack PE over libraries on the system when possible, as system libraries may change without warning.
```